pkg/http/server.go

```diff
@@ -157,7 +157,7 @@ func (s *server) StartNonBlocking() error {
 
 	if s.config.EnableProfiling {
 		for _, apiListenAddress := range s.config.APIListenAddresses {
-			addr := net.JoinHostPort(apiListenAddress, strconv.Itoa(s.config.ProfilePort))
+			addr := apiListenAddress + ":" + strconv.Itoa(s.config.Port)
```

Comment:
why is this changed from ProfilePort ? please change back.

Reply:
See here: https://github.com/dapr/dapr/pull/5648#discussion_r1053738839